upgrade/Creating_hybrid_configurations.md

```diff
@@ -0,0 +1,43 @@
+# Upgrading management and managed nodes with different CSM versions using IUF
+
+For certain use cases, there may be a need to combine different recipes of CSM to bring up the
+management and managed nodes. For example, if there is need to use a feature supported for specific
+CPU architecture for managed nodes which is present in a different recipe of CSM than the
+version of CSM that will be run on management nodes.
+
+## Procedure
+
+1. Decide the recipes for management and managed nodes. For example, use recipe **`a.b.c`**,
+which contains CSM version `1.5.x` for managed nodes and use recipe **`x.y.z`**, which contains
+CSM version **`1.6.x`**, for management nodes.
+
+1. Ensure that the CSM cluster is operational with the lowest among the selected recipes for both
+management and managed nodes.
+
+Use the [Upgrade CSM and additional products with IUF](../operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md) procedure to upgrade CSM.
+In the above example, the upgrade would be to recipe **`a.b.c`**, which contains
+CSM version **`1.5.x`**, for both management and managed nodes.
+
+1. Go through the `product_vars.yaml` file of both the recipes and come up with a set
+of products and versions across the recipes which have to be upgraded for CSM version **`1.6.x`**.
```

Comment:
How do they know which products should be upgraded. Should be all products with a new version in the latest recipe?